news/fix-default-style-dict.rst

```diff
@@ -0,0 +1,13 @@
+**Added:** None
+
+**Changed:** None
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:**
+
+* `prompt_toolkit >= 2` will start up even if Pygments isn't present
```

Comment:
Should have double backticks

Reply:
In Soviet Russia, backticks double _you_

Reply:
Also, we should set up an RST checker that lives outside of CI to save ourselves the runs

Reply:
Yeah for sure!